README.md

```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+# MACS 20500: Homework 1  
+## James Zimmerman  
+
+  
+### Intro  
+Hi, I'm James Zimmerman! I'm a rising third year here at UChicago majoring in **Public Policy** and in **Economics**.
+
+### Facts About Me  
+  + I'm from **New York City**, living my entire life in the same apartment before leaving for college.  
```

Comment:
Nice uses of emphasis through the document to highlight the really important takeaways you want the reader to identify with.